### PR TITLE
Natalia Lypovyk: Basic Math Operations

### PR DESCRIPTION
**Pull request by natalia-lypovyk**

### CODE REVIEW

File: src/basicOp.js [hunk 1]
```diff
@@ -25,6 +25,21 @@
  */
 function basicOp(operation, value1, value2) {
   // write code here
```
Comment (danheim) [inline src/basicOp.js]: Remove this line

File: src/basicOp.js [hunk 2]
```diff
@@ -25,6 +25,21 @@
...
+    result = value1 - value2;
+  } else if (operation === '*') {
+    result = value1 * value2;
+  } else if (operation === '/') {
+    result = value1 / value2;
+  } else {
+    result = 0;
+  }
+
+  return result;
```
Comment (danheim) [inline src/basicOp.js]: Do you need `result`?
```
if (condition) {
  return value1 + value2;
}
```

**Review by danheim — CHANGES_REQUESTED** [review]

File: src/basicOp.js [hunk 3]
```diff
@@ -24,7 +24,17 @@
...
+    return value1 + value2;
+  } else if (operation === '-') {
+    return value1 - value2;
+  } else if (operation === '*') {
+    return value1 * value2;
+  } else if (operation === '/') {
+    return value1 / value2;
+  } else {
+    return 0;
+  }
```
Comment (danheim) [inline src/basicOp.js]: remove `else`

**Review by danheim — CHANGES_REQUESTED** [review]

File: src/basicOp.js (hunk 3 above)
Comment (natalia-lypovyk) [reply src/basicOp.js]: removed

File: src/basicOp.js [hunk 4]
```diff
@@ -24,7 +24,17 @@
...
+    return value1 + value2;
+  } else if (operation === '-') {
+    return value1 - value2;
+  } else if (operation === '*') {
+    return value1 * value2;
+  } else if (operation === '/') {
+    return value1 / value2;
+  }
+
+  return 0;
```
Comment (danheim) [inline src/basicOp.js]: use switch case
https://mate-academy.github.io/style-guides/javascript.html#comparison--switch-blocks

File: src/basicOp.js [hunk 5]
```diff
@@ -24,7 +24,27 @@
...
+      return value1 * value2;
+    }
+
+    case '/': {
+      return value1 / value2;
+    }
+
+    default: {
+      return 0;
+    }
```
[the remark this answers is not in the document]
Comment (natalia-lypovyk) [reply src/basicOp.js]: removed

**Review by danheim — APPROVED** [review]

**Pull request closed without merging**